verigreen-collector-webapp/src/main/webapp/controllers.js

```diff
@@ -306,15 +309,18 @@ app.controller('ctrlRead', ['$scope','$filter','$http', 'ModalService', 'sharedP
 
 		$scope.filteredItems = $filter('filter')($scope.items, function(item) {
 			for (var attr in item) {
```

Comment:
@AndreeaDora you might want to try changing this to a more reliable loop construct for object *attributes* as we talked about a couple of weeks back. Please take a look at this: https://docs.angularjs.org/api/ng/function/angular.forEach